tmt/steps/prepare/ansible.py

```diff
@@ -70,4 +70,4 @@ def go(self, guest):
         # Apply each playbook on the guest
         for playbook in self.get('playbook'):
             self.info('playbook', playbook, 'green')
-            guest.ansible(playbook)
+            guest.ansible(playbook, self.get('extra-args'))
```

Comment:
Please, add a new option `--extra-vars` as well. Also, include the new key in the list of supported `_keys` at the top of the class definition.

Reply:
I added `extra-vars` as a "placeholder". There is an option `-e` in `ansible-playbook` that can be used within `extra-args` argument. I can try to implement new method like `_ansible_extra_vars` that will use `-e` option. Something like:
```
    @staticmethod
    def _ansible_extra_vars(extra_vars):
        """ Prepare additional variables for ansible-playbook"""
        return '' if extra_vars is None else f"-e {str(extra_vars)}"
```

Reply:
Oh, I'm sorry, have a typo in my comment. It should have been `--extra-args`. I just meant to add the click.option for the next config key [here](https://github.com/psss/tmt/blob/a0c7c28d335bda7484dcf626ce149474c413162d/tmt/steps/prepare/ansible.py#L49). Sorry for the confusion. Let's not add vars in this pull request.